netty/src/main/java/io/grpc/netty/NettyClientStream.java

```diff
@@ -80,11 +78,11 @@
       TransportState state, MethodDescriptor<?, ?> method, Metadata headers,
       Channel channel, AsciiString authority, AsciiString scheme, AsciiString userAgent,
       StatsTraceContext statsTraceCtx) {
-    super(new NettyWritableBufferAllocator(channel.alloc()), statsTraceCtx);
+    super(new NettyWritableBufferAllocator(channel.alloc()), statsTraceCtx, headers,
```

Comment:
wrap line to keep arguments on the same line.   That or do an argument per line.